app/models/saved_claim_group.rb

```diff
@@ -21,6 +21,13 @@ class SavedClaimGroup < ApplicationRecord
 
   scope :child_claims_for, ->(parent_id) { where(parent_claim_id: parent_id).where.not(saved_claim_id: parent_id) }
 
+  # Claim submission statuses
```

Comment:
might be good to have these in a STATUS constant so you can call `.freeze` on them so they don't get changed by accident in other parts of the code